Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            const int personCount = 2;
+
+            for (int i = 0; i < personCount; i++)
+            {
+                string firstName = GetString("Input your first name: ");
+                string lastName = GetString("Input your last name: ");
+                int age = GetInt("Input your age: ");
+                float height = GetFloat("Input your height in m: ");
+                float weight = GetFloat("Input your weight in kg: ");
+
+                float bmi = CalculateBmi(weight, height);
+
+                Console.WriteLine($"{firstName} {lastName} is {age} years old, his weight is {weight} kg and his height is {height} cm. His BMI is {bmi}");
```

Comment:
Please move this out ot a function- ProcessBmi() or some better way.

Reply:
But if I put it into a separate function, then I either have to put all the variables into global scope for them to be accessible inside the function (for the Console.WriteLine to use), or the function call will have to have a ton of parameters, which, as you mentioned, shouldn't be more that 2 (ideally).

Reply:
I would really appreciate if you could elaborate on how I should proceed to moving the code to calculate BMI into a separate function, as with my current understanding of ways I can think of to do that would involve braking the concepts you have mentioned in the lecture for this topic.
Thank you.